bootstrap/events/consumer/streams.go

```diff
@@ -44,34 +44,34 @@ func (es *eventHandler) Handle(ctx context.Context, event events.Event) error {
 
 	switch msg["operation"] {
 	case thingRemove:
-		rte := decodeRemoveThing(msg)
+		rte := decodeRemoveClient(msg)
 		err = es.svc.RemoveConfigHandler(ctx, rte.id)
 	case thingConnect:
```

Comment:
The value of thingConnect is thing.connect , this should be changed to client.connect